### PR TITLE
Removed stale Teku information

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description

- Updated testnet instructions to connect to Pyrmont
- Removed Altona network
- Removed non-standard REST API content

### CODE REVIEW

File: docs/HowTo/Get-Started/Connect-To-Testnet.md [hunk 1]
```diff
@@ -147,7 +145,7 @@ starting Teku.
 
 ## Finding help
 
-* The `#medalla` channel on the Ethereum Foundation Discord. [Invite link](https://discord.gg/zyZXUN7)
```
Comment (NicolasMassart) [inline docs/HowTo/Get-Started/Connect-To-Testnet.md:150]: I think the previous invite is still fine

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -745,10 +745,9 @@ Possible values are:
 | Network   | Chain   | Type        | Description                                                         |
 |-----------|---------|-------------|---------------------------------------------------------------------|
 | `mainnet` | Eth 2.0 | Production  | Main network.                                                       |
-| `altona`  | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
-| `medalla` | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
+| `medalla` | Eth 2.0 | Test        | Multi-client testnet.                                               |
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md:748]: Should we add something like "retired" in front of this testnet?

Comment (ajsutton) [reply docs/Reference/CLI/CLI-Syntax.md:748]: I think we could just remove medalla at this point.  The network is rapidly being abandoned and we'll likely shutdown our nodes in the next week or two anyway.

**Review by ajsutton — APPROVED** [review]
LGTM.

**Pull request merged**